.github/workflows/push-api-docs.yaml

```diff
@@ -0,0 +1,52 @@
+# When we push a new tag to the repository, open a Pull Request to synchronize
+# the API reference documentation with the "docs" repository.
+
+name: "Push API docs"
+
+concurrency:
+  group: "ci-${{ github.ref_name }}"
+  cancel-in-progress: true
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  push-api-docs:
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: "Checkout 'fabric' repository"
+        uses: "actions/checkout@v4"
+        with:
+          fetch-depth: 0
+          persist-credentials: "false"
+
+      - name: "Checkout 'docs' repository"
+        uses: "actions/checkout@v4"
+        with:
+          repository: "githedgehog/docs"
+          path: "project_documentation"
+          persist-credentials: "false"
+
+      - name: "Copy API reference from 'fabric' to 'docs'"
+        run: |
+          cp docs/api.md project_documentation/docs/reference/api.md
```

Comment:
We'd need to sync profiles too, but we can add it later

Reply:
Ah, yes. Let's do as a follow-up.

https://github.com/githedgehog/docs/pull/106, albeit with a source from a different repo, is a candidate, too.